pkg/asset/installconfig/platformprovisioncheck.go

```diff
@@ -160,6 +160,11 @@ func (a *PlatformProvisionCheck) Generate(dependencies asset.Parents) error {
 			return err
 		}
 
+		err = powervsconfig.ValidatePERAvailability(client, ic.Config)
+		if err != nil {
+			return err
+		}
+
```

Comment:
Doesn't need to be addressed in this PR, but maybe `powervs` can follow the pattern of the other platforms and use a `ValidateForProvisioning` function that encapsulates all the different validations needed.